qiskit_experiments/experiment_data.py

```diff
@@ -108,6 +109,16 @@ def job_ids(self) -> List[str]:
         """
         return list(self._jobs.keys())
 
+    @property
+    def completion_times(self) -> Dict[str, datetime]:
```

Comment:
Once `StoredData` is merged, this will be the proper location of the `completion_times` property

Reply:
Thanks for pointing this out @yaelbh . Which PR is `StoredData`?

Reply:
#113 and its continuation #115